gcloud/streaming/transfer.py

```diff
@@ -35,51 +34,99 @@
 RESUMABLE_UPLOAD = 'resumable'
 
 
+_DEFAULT_CHUNKSIZE = 1 << 20
```

Comment:
I see a constant like`1048576` as a "magic number", very easy to mistype without ever catching the problem.  The bit-shifted form is way harder to mistype, because it makes it clear what the number *means*.  For instance, I would way rather see:

```python
DAY_SECONDS = 24 * 60 * 60
```

than

```python
DAY_SECONDS = 86400
```